File: Latihan_A/Processor.cpp
class Processor
{
    private:
        // atribut
        string name;
        int price;

    public:
        // constructor
        Processor(){}
        Processor(string name, int price) {
            this->name = name;
            this->price = price;
        }

        // method set untuk name dan price
        void setName(string name) {
            this->name = name;
        }
        void setPrice(int price) {
            this->price = price;
        }

        // method get untuk name dan price
        string getName() {
            return this->name;
        }
        int getPrice() {
            return this->price;
        }

        // method output untuk Processor
        void printProcie() {
            cout << "Processor" << endl;
            cout << "-------------" << endl;
            cout << "Name        : " << this->name << endl;
            cout << "Price       : " << "Rp. " << this->price << endl;
        }

        // destructor
        ~Processor(){}
};

File: Latihan_B/python/Processor.py
class Processor:
    # atribut private
    __name = ""
    __price = 0

    # constructor
    def __init__(self, name="", price=0):
        self.__name = name
        self.__price = price

    # method set untuk name dan price
    def setName(self, name):
        self.__name = name
    def setPrice(self, price):
        self.__price = price

    # method get untuk name dan price
    def getName(self):
        return self.__name
    def getPrice(self):
        return self.__price
    
    # method untuk output Processor
    def printProcie(self):
        print("Processor")
        print("-------------")
        print("Name        : " + self.getName())
        print("Price       : " + "Rp. " + str(self.getPrice()))